order.go

```diff
@@ -262,6 +262,30 @@ type ShippingLines struct {
 	TaxLines                      []TaxLine        `json:"tax_lines,omitempty"`
 }
 
+// UnmarshalJSON custom unmarshaller for ShippingLines implemented to handle requested_fulfillment_service_id being
+// returned as json numbers or json nulls instead of json strings
+func (sl *ShippingLines) UnmarshalJSON(data []byte) error {
```

Comment:
Do we need any changes on how we marshal it back to JSON?

Reply:
Good question, I don't think we'd have problems when marshalling `ShippingLines` back to JSON because the change I'm proposing is scoped only to `ShippingLines.UnmarshalJSON` and since `ShippingLines` does not implement `MarshalJSON` and all its fields have primitive types it should work the same way it was working before.

Reply:
Sounds good. I believe Shopify should know how to deal with string values in this field. We can revisit it late if it create any hurdle for shipping apps/plugins, which are using integer IDs.